component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/EmailOtpServiceImpl.java

```diff
@@ -65,6 +65,7 @@
 public class EmailOtpServiceImpl implements EmailOtpService {
 
     private static final Log log = LogFactory.getLog(EmailOtpServiceImpl.class);
+    private static final boolean showFailureReason = EmailOtpServiceDataHolder.getConfigs().isShowFailureReason();
```

Comment:
should be in caps

Reply:
fixed with 3f45e923373f42d33ca282e1ee7fad51fa57f8c3